src/lib/viewers/box3d/model3d/Model3DViewer.js

```diff
@@ -334,7 +349,10 @@ class Model3DViewer extends Box3DViewer {
     handleReset() {
         super.handleReset();
 
-        this.isAnimationPlaying = false;
+        this.setAnimationState(false);
+        this.handleShowGrid(true);
```

Comment:
Does this interfere with the non-React controls at all?

Reply:
It shouldn't because it's essentially the same as what's being done below on L361-365. But now that you mention it, this can probably be simplified

Reply:
Tried to remove the controls calls but there's some hidden magic to set the defaults so I'll wait til we migrate over fully and just delete it then